src/main/java/com/endava/soa_5ed/task1_basics/Ex4.java

```diff
@@ -0,0 +1,41 @@
+package com.endava.soa_5ed.task1_basics;
+
+import java.util.Scanner;
+
+public class Ex4 {
+//    Write 3 methods that calculates sum of numbers from 0 to a given number.
+//            (1) Use for loop
+//            (2) Use while loop
+//            (3) Use do while loop
+
+    //1st method - for loop
+    int calc1(int num){
```

Comment:
try to rename this and the following 2 methods in such a way so you would not need the comments above each method

Reply:
done.